test/unittests/test_cpp.cpp

```diff
@@ -89,24 +91,24 @@ TEST(cpp, std_maps)
 {
     std::map<evmc::address, bool> addresses;
     addresses[{}] = true;
-    ASSERT_EQ(addresses.size(), 1);
+    ASSERT_EQ(addresses.size(), size_t{1});
```

Comment:
It's annoying to do the cast every time, is it fixing the warning?
Do you not like `1u` better?

Reply:
Somehow combination of GTest 1.8.1 and Visual Studio 2015 produces warnings about signed/unsigned comparison. Hopefully, we could drop VS 2015 at some point.

It could have been `1u`, I'm just not used to using them because `ul` / `unsigned long` has different size on VS and GCC/clang.